main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -239,21 +237,20 @@ object NameError {
     * @param loc  the location of the class name.
     */
   case class UndefinedNativeClass(name: String, loc: SourceLocation) extends NameError {
-    def summary: String = s"Undefined class."
+    def summary: String = s"Undefined Java class '$name'."
 
     def message(formatter: Formatter): String = {
       import formatter._
       s"""${line(kind, source.format)}
-         |>> Undefined class '${red(name)}'.
+         |>> Undefined Java class '${red(name)}'.
          |
-         |${code(loc, "undefined class.")}
+         |${code(loc, "undefined Java class.")}
          |""".stripMargin
     }
 
-    /**
-      * Returns a formatted string with helpful suggestions.
-      */
-    def explain(formatter: Formatter): Option[String] = None
+    def explain(formatter: Formatter): Option[String] = Some({
+      "Flix cannot find the Java class. Maybe there is a typo?"
```

Comment:
I think there is also a decent chance that the user forgot to include the jar when compiling.